ext/LegendSpecFitsRecipesBaseExt.jl

```diff
@@ -6,7 +6,7 @@ using RecipesBase
 using Unitful, Format, Measurements, LaTeXStrings
 using Measurements: value, uncertainty
 using StatsBase, LinearAlgebra
-
+using BAT 
```

Comment:
Could you replace the `BAT` dependency by a dependency on `KernelDensity`?
This would also require adding `KernelDensity` in the `Project.toml` with `[compat]` entry `"0.5, 0.6"`

Reply:
We need `BAT` anyway in `LegendSpecFits`. Isn't it more complicated to also add `KernelDensity`?

Reply:
This would be the „clean“ way, to also have version control over KernelDensity.
As Oli pointed out, maybe BAT might get rid of that dependency in the near future, then it would be nice to have it separated here.